OracleWebCenterSites/README.md

```diff
@@ -94,20 +94,26 @@ Sign in to [Oracle Container Registry](https://container-registry.oracle.com). C
 ### C. To download Oracle WebCenter Sites Docker files and binary file.
 
 1. Clone the GitHub repository. URLs for cloning: 
-
  - HTTPS:[ https://github.com/oracle/docker-images.git](https://github.com/oracle/docker-images.git) 
-
  - SSH: [git@github.com:oracle/docker-images.git](git@github.com:oracle/docker-images.git).
-
- The repository contains Docker files and samples to build Docker images for Oracle products and open source projects.
-
+    
+    The repository contains Docker files and samples to build Docker images for Oracle products and open source projects.
 2. Download Oracle WebCenter Sites 12c R2 12.2.1.3 binary from [Oracle Technology Network](http://www.oracle.com/technetwork/middleware/webcenter/sites/downloads/index.html).
-
-3. Save the Oracle WebCenter Sites 12.2.1.3 binary at this location: `../docker-images-master/OracleWebCenterSites/dockerfiles/12.2.1.3/`.
+3. Save the Oracle WebCenter Sites 12.2.1.3 binary at this location: `../docker-images/OracleWebCenterSites/dockerfiles/12.2.1.3/`.
+4. Download WebCenter Sites: Automated Installation Scripts binary from [https://support.oracle.com](https://support.oracle.com).
```

Comment:
This procedure should be documented as "For Existing Oracle Customers" but you need to also provide an alternative download location for non-Oracle users, i.e. people who have no access to My Oracle Support.